utils/build.ps1

```diff
@@ -739,6 +740,13 @@ function Get-PinnedToolchainRuntime() {
   return "$BinaryCache\toolchains\${PinnedToolchain}\PFiles64\Swift\runtime-development\usr\bin"
 }
 
+function Get-PinnedToolchainVersion() {
+  if (Test-Path variable:PinnedVersion) {
+    return $PinnedVersion
+  }
+  return "5.10.1"
+}
+
```

Comment:
I was going to have this query the compiler binary for the version, but it felt a bit silly as the version would be present in the path to `swiftc.exe`. I think there's some cleanup work to do for this, but it's out of scope for this PR.